README.md

```diff
@@ -1,2 +1,69 @@
-# rustls-symcrypt
-Glue between Rustls and rust-symcrypt 
+# SymCrypt Provider for Rustls
+
+This crate provides integration for using [rust-symcrypt](https://github.com/microsoft/rust-symcrypt) cryptographic functionalities with  [rustls](https://github.com/rustls/rustls), by implementing the required traits specified by `rustls`.
+
+## Status and Platform Support:
+- Currently works with: `rustls = { version = "0.23.0", features = ["ring", "tls12", "std"], default-features = false }`
```

Comment:
>"ring" [](http://example.com/codeflow?start=68&length=6)

Is Ring still needed?

Reply:
Ring is needed for now, im planning on replacing this ring dependency once we finish adding RSA into rust-symcrypt